src/components/volunteer-enroll/UpdateCriminalCheck.tsx

```diff
@@ -0,0 +1,148 @@
+                    </Text>
+                </Box>
+                <Box>
+                    <Text
+                        pb="5px"
+                        align="left"
+                        fontWeight="700"
+                        mt="50px"
+                        fontSize="22px"
+                    >
+                        Your criminal record check has expired!
+                    </Text>
+                    <Text pb="5px" align="left" mt="30px">
+                        As volunteering in our programs involves working closely
+                        with children and vulnerable persons, we ask that all
+                        our volunteers get a Criminal Record Check completed at
+                        their local police station or the RCMP office. The price
+                        of the CRC will be waived with a letter provided from
+                        SDC.
+                    </Text>
+                    <Text pb="5px" align="left" mt="30px">
+                        Also, please note that the MPM/JELIC is an IN-PERSON
+                        math program. If you apply to volunteer for this
+                        program, pleasure ensure that you are aware that it is
+                        an in-person program and are able to attend the classes
+                        at Richmond Quantum Academy (6650-8181 Cambie Rd,
+                        Richmond, BC V6X 3X9).
+                    </Text>
+                </Box>
+                <Box>
+                    <Text
+                        pb="5px"
+                        align="left"
+                        fontWeight="700"
+                        mt="50px"
+                        fontSize="22px"
+                    >
+                        Uploading your Criminal Record Check
+                    </Text>
+                    <OrderedList pb="5px" align="left" mt="30px">
+                        <ListItem>
+                            Under My Account {">"} Criminal Record Check
+                            generates a volunteer letter from SDC
+                        </ListItem>
+                        <ListItem>
+                            Use the provided letter to obtain a criminal record
+                            check at the local police station or RCMP office
+                        </ListItem>
+                        <ListItem>
+                            Upload a copy of the document to your SDC account
+                        </ListItem>
+
+                        <ListItem>
+                            Once you've submitted your document(s), keep an eye
+                            out for the approval status from SDC!
+                        </ListItem>
+                    </OrderedList>
+                </Box>
+                <Box>
+                    <Link
+                        href={`/document-upload?type=criminal-check&redirect=volunteer/enrollment?classId=${classInfo.id}%26page=1`}
```

Comment:
For href, we always want to specify absolute path otherwise i18n subpaths won't be happy. We can also make the page redirect a bit more robust
```suggestion
                        href={`/document-upload?type=criminal-check&redirect=/volunteer/enrollment?classId=${classInfo.id}%26page=1`}
```